packages/third-parties/bullmq/src/config/config.ts

```diff
@@ -1,13 +1,51 @@
-import {type ConnectionOptions, type DefaultJobOptions} from "bullmq";
+import {type ConnectionOptions, type DefaultJobOptions, QueueOptions, WorkerOptions} from "bullmq";
```

Comment:
```suggestion
import {type ConnectionOptions, type DefaultJobOptions, type QueueOptions, type WorkerOptions} from "bullmq";
```

Best make this also type imports I think.